src/auth/registration/AccountChoice.tsx

```diff
@@ -1,175 +1,130 @@
-      fill="#687485"
-    />
-  </svg>,
-  "Team",
-);
+import { useState } from "react";
+import logo from "../../../brave-icon-release-color.svg";
+import { useTrackMatomoPageView } from "hooks/useTrackWithMatomo";
+import { useIsMobile } from "hooks/useIsMobile";
+import { AuthContainer } from "auth/views/components/AuthContainer";
 
 interface ChoiceOptions {
   title: MessageDescriptor;
-  icon: any;
-  description: MessageDescriptor;
-  points: MessageDescriptor[];
-  value: AccountSetup;
-  minSpend: string;
+  value: string;
 }
 
 export function AccountChoice() {
+  const isMobile = useIsMobile();
+  useTrackMatomoPageView({
+    documentTitle: `Ads Account Choice`,
+  });
+
+  const [accountType, setAccountType] = useState<string>();
   const buttons: ChoiceOptions[] = [
     {
-      title: msg`Self-service`,
-      icon: <Pointer sx={{ height: "1.5em", width: "1.5em" }} />,
-      description: msg`For small to mid-sized businesses to create and manage their Brave Ads campaigns independently.`,
-      points: [msg`Notification ads`, msg`Newsfeed ads`],
-      value: "self",
-      minSpend: "500",
+      title: msg`Browser ads`,
+      value: "browser",
     },
     {
-      title: msg`Managed service`,
-      icon: <Team sx={{ height: "1.5em", width: "1.5em" }} />,
-      description: msg`For enterprise and agency businesses looking for bespoke ad solutions via dedicated Brave Ads representatives.`,
-      points: [
-        msg`Search keyword ads`,
-        msg`New tab takeover`,
-        msg`Notification ads`,
-        msg`Newsfeed ads`,
-      ],
-      value: "managed",
-      minSpend: "10,000",
+      title: msg`Search ads`,
+      value: "search",
     },
   ];
 
   return (
-    <List
-      sx={{ display: "flex", flexDirection: "row", gap: 3, flexWrap: "wrap" }}
-    >
-      {buttons.map((b) => (
-        <AccountItemButton key={`account_item_button_${b.value}`} {...b} />
-      ))}
-    </List>
+    <AuthContainer>
+      {!isMobile && <div style={{ width: 500 }} />}
+      <img src={logo} height={65} width={100} style={{ marginBottom: 15 }} />
+      <Typography variant="h5">
+        <Trans>Choose campaign type</Trans>
+      </Typography>
+      <Link
+        underline="none"
+        variant="caption"
+        href="https://ads-help.brave.com/ad-placements/brave-browser/notification"
```

Comment:
The content on this url just talks about notification ads, which is confusing. Did you mean https://ads-help.brave.com/category/ad-placements?

Though the content on that page clearly says "Available only via Managed Service buys" which will no longer be true - do we need to synchonise an update to that content with this release?

Reply:
I think: https://ads-help.brave.com/category/brave-browser should suffice, it is a new addition